AppAuth-JS/built/errors_test.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork